src/Stryker.Core/Stryker.Core.UnitTest/Options/FilePatternTests.cs

```diff
@@ -45,10 +46,11 @@ public void IsMatch_should_match_glob_pattern(string file, string glob, bool isM
         public void IsMatch_should_match_textSpans(string spanPattern, int spanStart, int spanEnd, bool isMatch)
         {
             // Arrange
-            var sut = FilePattern.Parse("*.*" + spanPattern);
+            var mutantSpan = new MutantSpan(spanStart, spanEnd);
+            var pattern = new CsharpFileLeaf().Parse(new ExcludableString("*.*" + spanPattern));
 
             // Act
-            var result = sut.IsMatch($"test.cs", TextSpan.FromBounds(spanStart, spanEnd));
```

Comment:
Also, good sign - we don't need to pass random file name data to perform span arithmetics.